symengine/cwrapper.cpp

```diff
@@ -563,6 +565,50 @@ size_t vecbasic_size(CVecBasic *self)
     return self->m.size();
 }
 
+// C wrapper for Matrix
+
+struct CDenseMatrix {
+    SymEngine::DenseMatrix m;
+};
+
+CDenseMatrix *dense_matrix_new()
+{
+    return new CDenseMatrix;
+}
+
+void dense_matrix_free(CDenseMatrix *self)
+{
+    delete self;
+}
+
+void dense_matrix(CDenseMatrix *s)
```

Comment:
@rajithv, you don't need this function right? You can use `dense_matrix_new()`, right?